activation/activation.go

```diff
@@ -111,7 +111,7 @@ type Builder struct {
 type positioningAtxFinder struct {
 	finding sync.Mutex
 	found   *struct {
-		id         types.ATXID
+		id         *types.ATXID
```

Comment:
Personally I find this specific block to be not a good example of doing things - using anonymous structs in such cases is something I'd generally like to avoid. I found _a lot_ of occurences where code was written such that the assumption that an uninitialized variable/field defaults to an empty ATX ID. I personally object to this sort of working paradigm, preferring the use of constructors with arguments that set field values, and concrete types. It reduces the mental taxation of reading code and understanding what it's doing. Having a default value doesn't really help from my perspective, I'd rather see the field being set explicitly always.